docs/hoon/twig/cen-call/sig-open.md

```diff
@@ -13,6 +13,10 @@ sort: 3
 :rap(p :make(q +6 r))
 ```
 
+```
+=<(p %=(q +6 r))
+```
+
```

Comment:
This is another example of the ambiguity in the expansions. I puzzled over this one for quite a while.

I started with this

```
%~($ |=(a/@ +(a)) 2)
```

And then tried this

```
 =<($ %=(|=(a/@ +(a)) +6 2))
```

which doesn't compile, and can't even be typed in dojo. Eventually, I reread the twig description, so that `p` is a `wing`, and got this to work:

```
 =<($ =+(q=|=(a/@ +(a)) %=(q +6 2)))
```

I thought about making the case for that expansions, till I saw the list unwrapping in the `?` twigs ...

/cc @ohAitch